app/src/main/java/edu/usf/sas/pal/muser/model/Event.kt

```diff
@@ -1,24 +1,23 @@
 package edu.usf.sas.pal.muser.model
 
 /**
- * A model class that stores the actions performed on the song track. The actions like play, pause,
+ * A model interface that stores the actions performed on the song track. The actions like play, pause,
  * skip, repeat and seek are stored along with the nested [SongData] class.
  */
 
-data class Event
+interface Event
 
 /**
- * [event] - The event attribute includes events like play, pause, skip, repeat and seek.
  * [currentTimeMs] - Stores the timestamp of the action recorded.
  * [nanoTime] - Stores the  value of the running JVM's time source in nanoseconds.
  * [startTime] - Start time of the song.
  * [elapsedTime] - Elapsed time of the song when the event occurred.
  * [song] - Song on which the action was performed.
  */
-(val event:EventType,
- val currentTimeMs: Long,
- val nanoTime: Long,
- val startTime: Long,
- val elapsedTime: Long,
- val song: SongData
-)
\ No newline at end of file
+{
+    val currentTimeMs: Long
+    val nanoTime: Long
+    val startTime: Long
+    val elapsedTime: Long
```

Comment:
I'm still not convinced that the current `elapsedTime` get us what we need in terms of recording where in the track the user started and stopped it, as the current value looks like a clock time to me (e.g., "Aug 26 3:00pm"). 

So if you play the track at Aug 26 3:00pm, and then pause it at 3:02pm, and then resume playing it at 3:05pm, the `elapsedTime` value is 3:05pm (?). But that doesn't tell us that the user resumed playing the track 2 minutes into the track. This value is also redundant to `currentTimeMs`, which we can use to look at each event and calculate the duration between events.

I think we need more of a `elapsedTrackTime` that tells us how many seconds into the track an action occurred, like "120 seconds". So we know that a user played the track and started it at 120 seconds in, and then paused it again at 130 seconds in.

Reply:
Similar for `startTime` - how is that different from `currentTimeMs`?

Reply:
Yes - You're right. the `elapsedTime` is just getting the current time and doesn't really work as we think it should. I guess we'll have to calculate and store `trackPositionSec` as you suggested.

Reply:
Also, `startTime` updates and stores the timestamp when a track is played (not paused) by the user. I guess the attribute helps to keep track of when a particular song was last played?

Reply:
Can we get rid of `elapsedTime` and `startTime`? For our work, if we're tracking the `currentTimeMs` and `nanoTime` for each event, I don't see how these other variables add any additional info.

Reply:
Yes - I agree.

Reply:
Per discussion today, let's change `elapsedTime` to `seekPositionMs`.